components/mediation-ui/mediators-ui/org.wso2.carbon.mediator.call.ui/src/main/resources/web/call-mediator/update-mediator.jsp

```diff
@@ -79,6 +79,9 @@
         callMediator.setEndpoint(ep);
     }
 
+    boolean blocking=Boolean.parseBoolean(request.getParameter("mediator.call.blocking"));
```

Comment:
This is fixed in the latest code.
callMediator.setBlocking(Boolean.parseBoolean(request.getParameter("mediator.call.blocking")));